src/Inkluzitron/Modules/BdsmTestOrg/BdsmModule.cs

```diff
@@ -297,30 +285,36 @@ public async Task ProcessQuizResultSubmissionAsync([Name("odkaz")] string link)
             var responseData = await response.Content.ReadAsStringAsync();
             var testResult = JsonConvert.DeserializeObject<Result>(responseData);
 
-            var testResultDb = new BdsmTestOrgQuizResult
+
+            var user = await PointsService.GetOrCreateUserEntityAsync(
```

Comment:
PointsService by neměla být zodpovědná za vytváření uživatelů. Minimálně by to chtělo nějak přejmenovat.